### PR TITLE
Update de.yml

### PR DESCRIPTION
**Pull request by zweiundvierzig**
translated some untranslated things, some minor grammatical corrections

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -645,7 +645,7 @@ de:
   label_change_status: "Statuswechsel"
   label_change_view_all: "Alle Änderungen anzeigen"
   label_changes_details: "Details aller Änderungen"
-  label_changeset_plural: "Changesets"
+  label_changeset_plural: "Änderungs-Sets"
```
Comment (ulferts) [inline config/locales/de.yml]: I would stick to Changesets, might be "Denglisch" but everybody knows what is meant.

**Comment by ulferts** [comment]
Except from the "Änderungs-Sets" I like this. Thanks for providing.

**Comment by zweiundvierzig** [comment]
I think you are right: "Changesets" is more usual than "Änderungs-Sets" :-)

**Comment by ulferts** [comment]
Could you please revert that in your PR, than? Once that is through I can merge.

**Comment by zweiundvierzig** [comment]
...reverted!

**Pull request merged**